### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by joar**

### CODE REVIEW

File: fish.py [hunk 1]
```diff
@@ -290,6 +290,7 @@ class Bird(SwimFishTimeSync, BirdLook):
 if __name__ == "__main__":
     import signal
     import optparse
+    from __future__ import print_function
```
Comment (habnabit) [inline fish.py]: This must be the first statement in the file.

Comment (joar) [inline fish.py]: Fixed in ae76d32, not tested

**Comment by lericson** [comment]
Pretty sure Python 3.x support was added on the 16th of September, 2014 in 08c1f5a00c6ccf1fe4398f71bc408773d3f0ff44.

**Comment by joar** [comment]
"OK" :fried_shrimp: 

I see "2to3" is used. I probably ran it from a git clone, so 2to3 was never invoked. I'll try pip-installing it next time.

**Comment by lericson** [comment]
Is that a shrimp?

Always pippa.

**Pull request closed without merging**